src/pages/Account/Account.scss

```diff
@@ -0,0 +1,75 @@
+@use "../../styles/variables" as *;
+
+.account-page {
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 1.5rem;
+  }
+  &__text-content {
+    margin: 1rem 3rem;
+    padding: 0 1rem;
+
+    &--box {
+      width: 18rem;
+      height: 2px;
+      background-color: $color-grey;
+      margin-right: 1rem;
+    }
+
+    &--first-name {
+      font-size: 1.2rem;
+      margin-bottom: 0.1rem;
+    }
+
+    &--last-name {
+      font-size: 1.2rem;
+      margin-bottom: 0.1rem;
+    }
+
+    &--name {
+      font-size: 0.9rem;
+      margin-bottom: 1rem;
+      margin-top: 0;
+    }
+
+    &--email {
+      font-size: 1.2rem;
+      margin-bottom: 0.1rem;
+    }
+
+    &--email-address {
+      font-size: 0.9rem;
+      margin-bottom: 1rem;
+      margin-top: 0;
+    }
+  }
+}
+
+.sign-out-button {
+  position: absolute;
+  top: 80%;
+  left: 9%;
+}
+
+.input-box {
+  &__label {
+    padding: 0.2rem 0 0.3rem 0;
+    font-size: 1.2rem;
+  }
+  &__input {
+    padding: 0 0 1rem 0;
+    &--box {
+      font-size: 0.8rem;
+      line-height: 20px;
+      width: 100%;
+      padding: 0;
+      &:active,
+      &:focus {
+        outline: none;
+      }
```

Comment:
take a look at the padding here, it looks a bit strange the input is miles away from the underline. Not sure if it's this scss or somewhere else